css/ampstart-base/base-page.css

```diff
@@ -15,7 +15,22 @@
  */
 @import 'base-page-vars';
 
-* {
+amp-sidebar,
+div,
+section,
+input,
+textarea,
+select,
+label,
+button,
+.btn,
+.border,
+.border-top,
+.border-right,
+.border-bottom,
+.border-left,
+.ampstart-btn,
+.ampstart-input {
   box-sizing: border-box;
```

Comment:
This is an intentional reset. Also remember we should almost never override bass  classes (especially not like this)